dmoj/problem.py

```diff
@@ -159,6 +159,19 @@ def load_checker(self, name):
         self._checkers[name] = checker = load_module_from_file(os.path.join(self.root_dir, name))
         return checker
 
+    @property
+    def grader_class(self):
```

Comment:
How did you fix the mypy here?

Reply:
My secret move: I tacked on a `# FIXME(tbrindus)` with `Any` :slightly_smiling_face: